lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2647,10 +3077,13 @@ int s1ap_mme_handle_enb_configuration_transfer(
     }
   }
 
-  message->procedureCode = S1ap_ProcedureCode_id_MMEConfigurationTransfer;
-  message->direction     = S1AP_PDU_PR_initiatingMessage;
+  pdu->choice.initiatingMessage.procedureCode =
+      S1ap_ProcedureCode_id_MMEConfigurationTransfer;
+  pdu->present = S1ap_S1AP_PDU_PR_initiatingMessage;
+  // Tricky- optimisation needed
```

Comment:
Do we need a ToDo comment here? This comment is not very clear on what needs to be done